src/frontend/src/common/queries/trees.ts

```diff
@@ -73,6 +75,7 @@ async function createTree({
       filter_start_date: startDate,
       filter_end_date: endDate,
       filter_pango_lineages: lineages,
+      filter_location_id: location?.id,
```

Comment:
@vincent-czi is this the correct payload?

Reply:
Not quite: there's no word "filter" on the location aspect, it should be **just `location_id`**. So replace `filter_location_id` here and in the `CreateTreePayload` interface with just `location_id`. (The ticket names around this work are kind of a misnomer being called tree location filter. All the trees we build have precisely one location associated with them, it's just that right now the user has no control over what that location is. So it's not really a filter so much as it's just a way to set a template arg that would otherwise get defaulted. That's the reasoning behind why I named it just `location_id`, if you're curious.)

Reply:
got it, that makes sense.  I was wondering about that.